content/docs/guides/python.md

```diff
@@ -102,6 +104,39 @@ print('Current time:', time)
 print('PostgreSQL version:', version)
 ```
 
+```python
+import os
+import asyncio
+import asyncpg
+from dotenv import load_dotenv
+
+async def main():
+   # Load .env file
+   load_dotenv()
+
+   # Get the connection string from the environment variable
+   connection_string = os.getenv('DATABASE_URL')
+
+   # Connect to the Postgres database
+   conn = await asyncpg.connect(connection_string)
```

Comment:
Not sure if we should do it, but we can recommend using pools instead of a single `conn`, since it's better for server applications.

https://magicstack.github.io/asyncpg/current/usage.html#connection-pools

Reply:
I think we may update both in this case?

Reply:
Sure, let's update both.

Reply:
Sounds good!